config/help/day3.md

```diff
@@ -0,0 +1,10 @@
+## Example Help for WODIN app
+
+Here is an example formula which can be included in markdown help for an app:
+
+$$\frac{dS}{dt}=-\beta \frac{SI}{N}+\nu I$$
+
+And here is an example of an inline maths expression: $((\\beta I dt)/N)$
+
+Here is an example image, which should be placed in the `help/config` folder: ![example image](/example_img.png)
```

Comment:
With the approach in the front end, will the image need the forward slash? if they will be relative paths, making the rewrite work so that this was a relative path (i.e.,  `![example image](example_img.png)`) would be closer to the normal markdown usage

Reply:
Current front end implementation does identify local images as starting with a forward slash, but it could check for not starting with https:// instead..